src/index.html

```diff
@@ -1,13 +1,107 @@
 <!doctype html>
-<html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+<html class="page"
+      lang="en">
+
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible"
+          content="ie=edge">
+    <title>Moyo header</title>
+    <link rel="stylesheet"
+          href="./styles/index.css">
+  </head>
+
+  <body class="page__content">
+    <header class="header">
+      <a class="logo-link"
+         href="#home">
+        <img src="images/logo.png"
+             alt="Moyo logo">
+      </a>
+      <nav class="main-nav">
+        <ul class="main-nav-list">
+          <li class="main-nav-item">
+            <a class="main-nav-item__link is-active"
+               href="#">Apple</a>
```

Comment:
```suggestion
            <a 
              class="main-nav-item__link is-active"
              href="#"
            >
              Apple
            </a>
```
Please fix all such problems in code. Let's follow code style.